README.md

```diff
@@ -329,14 +329,67 @@ LIT_SERVER_API_KEY=supersecretkey python main.py
 Clients are expected to auth with the same API key set in the `X-API-Key` HTTP header.
 
 </details>
+
+
+<details>
+  <summary>Stream long responses</summary>
+
 &nbsp;
 
+`LitServer` can stream the outputs for LLMs or any model of your choice.
+
+To enable streaming, you need to implement `LitAPI.predict` and `LitAPI.encode_response` as a generator (a Python
+function which yields output) and set `LitServer(..., stream=True)`.
+
+For example, running the API server for an LLM with streaming:
+
+```python
+from typing import Generator
+
+from litserve.api import LitAPI
+
+from litserve import LitServer
+from pydantic import BaseModel
+
+
+class PromptRequest(BaseModel):
+    prompt: str
+
+
+class SimpleStreamAPI(LitAPI):
+    def setup(self, device) -> None:
```

Comment:
any recommendation on model here which is easy to get started with?

Reply:
Good question, we need to keep this minimal.

Instead of an LLM, can we just generate a sequence of random integers (1-10) as the model?

The other question is around `encode_response`: I think you should structure it as individual payloads (look at how OpenAI or ollama does it), we can't just return raw numbers as the payload.

Reply:
updated to return a json payload instead.